test/functionalTest/cases/0283_coap_support/coap_basic.test

```diff
@@ -31,9 +31,8 @@ proxyCoapStart
 
 --SHELL--
 
-valgrindSleep 2
+valgrindSleep 5
```

Comment:
At the end, all the proxyCoap stuff (code, makefile targets, tests, etc.) will be moved to an "archive" directory, so it is not a problem if this 0283_coap_support tests have problems.

NTC